betka/gitlab.py

```
@@ -520,15 +540,17 @@ def get_project_id_from_url(self):
         url = "https://gitlab.com/api/v4/projects"
         headers = {
             "Content-Type": "application/json",
-            "PRIVATE-TOKEN": self.betka_config["gitlab_api_token"].strip()
+            "PRIVATE-TOKEN": self.betka_config["gitlab_api_token"].strip(),
         }
         url_namespace = f"{self.config_json['gitlab_namespace']}/{self.image}"
         url = f"{url}/{url_namespace.replace('/', '%2F')}"
         logger.debug(f"Get project_id from {url}")
         ret = requests.get(url=f"{url}", headers=headers, verify=False)
```

Ruff (0.15.5)

[error] Probable use of `requests` call without timeout

(S113)